src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -198,6 +200,49 @@ def endAuction():
 
 This example should give you a sense of what Vyper contract syntax is like. For a more detailed description of the functions and variables, [see the docs](https://vyper.readthedocs.io/en/latest/vyper-by-example.html#simple-open-auction).
 
+## Yul and Yul+ {#yul}
+
+**Yul**
+- Intermediate language for Ethereum.
+- Supports EVM and eWASM, and it is designed to be a usable common denominator of both platforms.
+- Good target for high-level optimisation stages that can benefit all target platforms equally.
```

Comment:
I'm not following this but I am also not a developer. Will defer to @samajammin here, but we may want to explain what this means a little more.

Reply:
I'm also unclear on this @franzihei - mostly around what "target" is referring to.
- Does "Good target for high-level optimisation stages" == "Useful language for high-level optimisation stages"?
- Does "all target platforms"== "EVM and eWASM"?